.github/workflows/compare-benchmark.yml

```
@@ -113,23 +162,23 @@ jobs:
           echo "ver=$VER" >> "$GITHUB_OUTPUT"
 
       - name: Kill BASE services
-        if: steps.prver.outputs.ver_diff == 'true'
+        # if: steps.prver.outputs.ver_diff == 'true'
         run: |
           pkill -f "aztec.*--sandbox" || true                   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)